connection.go

```diff
@@ -97,11 +97,11 @@ func processSetStatement(ctx context.Context, c *fireboltConnection, query strin
 		return false, err
 	}
 
-	parameters := map[string]string{setKey: setValue}
-	if db, ok := c.parameters["database"]; ok {
-		parameters["database"] = db
-	}
-	_, err = c.client.Query(ctx, c.engineUrl, "SELECT 1", parameters, connectionControl{
+	// combine parameters from connection and set statement
+	combinedParameters := c.parameters
```

Comment:
Do we clone here or just copy the reference? Probably worth adding a test, that checks that we don't add the parameter if `select 1` has failed. Not sure if we have one

Reply:
Good spot, I assumed it will be copied, but reading up on it the maps are collections of references so the items are original ones. I will rework it.

Reply:
Done